benchmarks/inference-server/text-generation-inference/variables.tf

```diff
@@ -58,6 +58,17 @@ variable "gpu_count" {
   }
 }
 
+variable "max_concurrent_requests" {
+  description = "Max concurrent requests allowed for TGI to handle at once."
```

Comment:
Would be good to add to the description that TGI will drop all requests once it hits the max-concurrent-requests limit. Also, once we find a good default that works with the default benchmarking setup (1000 or whatever that number is), we should set that as default instead of 128.

Reply:
great idea, updated the description!

+1, will update the default post-experimentation :) added todo for now